### ISSUE TITLE
Not able to navigate to Polymer 2 elements

### ISSUE THREAD

**Issue by bahrus** [Polymer/vscode-plugin#53]
Maybe I'm missing something.

For this page:

https://github.com/bahrus/xtal/blob/master/tests/xtal-browser-window-test1.html

if I F12 on the xtal-browser-window tag, it says "No definition found."

The page does work as expected in chrome.

**Comment by rictic** [comment]
This looks like https://github.com/Polymer/polymer-analyzer/pull/540

Should be fixed in the next release.

**Comment by rictic** [comment]
As a workaround, try adding `/** @polymerElement */` above the class

**Comment by bahrus** [comment]
That didn't seem to work, but I moved the code to typescript, so that might be complicating things.  Speaking of which, do you know if someone is working on Polymer 2 type definitions?

**Comment by rictic** [comment]
Depending on how you're writing the typescript we may be able to analyze the javascript it emits, if it emits ES6.

We've got a simple proof of concept for generating definitions using the analyzer, not sure where it is right now.

**Comment by bahrus** [comment]
This is fixed now!  Really appreciate it.  This plug-in is amazing.  This fix takes me one giant step closer to doing my show and tell at work.

I should mention, as you know, that when I jump from the html file where the custom tag is referenced, to the source code for the custom element, it takes me to the JavaScript file, not the TypeScript file.  This would be sort of a problem were it not for some [additional plugins I've found](https://marketplace.visualstudio.com/search?term=switch&target=VSCode&category=All%20categories&sortBy=Relevance).

I was able to get the "Switcher" one working, and it is even thoughtful enough to take me to the same line number, which is usually pretty close to the equivalent code.

Might be something worth mentioning perhaps, or even including in the plug-in (at the risk of mission creep).
